spec/controllers/mixins/ems_common/angular_spec.rb

```diff
@@ -85,25 +85,25 @@
 
       it "returns connect options for openstack cloud default tab" do
         @params[:cred_type] = "default"
-        @ems_cloud_controller.instance_variable_set(:@_params, @params)
+        @ems_cloud_controller.params = @params
```

Comment:
@h-kataria @himdel these ivar sets will have to go throughout the test suite as they bypass creation of the new AC::Parameters object and don't test how it works in production.

See [here](https://github.com/rails/rails/blob/v5.0.7.2/actionpack/lib/action_controller/metal/strong_parameters.rb#L1059)

Reply:
Can you have someone go through the rest of the tests? There are 432 of these that don't evoke a failure but aren't testing how it works in production.  Thanks!

```
git grep -E "instance_variable_set.+\_params" | wc -l
     432
```

Reply:
Sure :) https://github.com/ManageIQ/manageiq-ui-classic/pull/5691